CONTRIBUTING.rst

```diff
@@ -1,13 +1,19 @@
 Contributing
 ============
 
-1.  **Please sign one of the contributor license agreements [below][6].**
-1.  [File an issue][9] to notify the maintainers about what you're working on.
-1.  [Fork the repo][10], develop and [test][11] your code changes, add docs.
-1.  Make sure that your commit messages clearly describe the changes.
-1.  [Send][12] a pull request.
+Here are some guidelines for hacking on `endpoints-management-python`_.
 
-Here are some guidelines for hacking on `google-endpoints-api-management`.
+-  Please **sign** one of the `Contributor License Agreements`_ below.
+-  `File an issue`_ to notify the maintainers about what you're working on.
+-  `Fork the repo`_; develop and `test your code changes`_; add docs.
+-  Make sure that your `commit messages`_ clearly describe the changes.
+-  `Make the pull request`_.
+
+.. _`Fork the repo`: https://help.github.com/articles/fork-a-repo
+.. _`forking`: https://help.github.com/articles/fork-a-repo
+.. _`commit messages`: http://chris.beams.io/posts/git-commit/
+
+.. _`File an issue`:
```

Comment:
That's intended.  This makes it an internal target that is linked to from some referencing text above.